hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go

```diff
@@ -1207,6 +1383,20 @@ prom-rules-verify: build-prom-spec-dumper
 
 `
 
+const metricsdocsMakefileFragment = `
+
+##@ Generate the metrics documentation
+.PHONY: generate-metricsdocs
+generate-metricsdocs: build-metricsdocs
+	mkdir -p $(shell pwd)/docs/monitoring
+	_out/metricsdocs > docs/monitoring/metrics.md
+
+##@ Build the metrics documentation
+.PHONY: build-metricsdocs
+build-metricsdocs:
+	go build -ldflags="${LDFLAGS}" -o _out/metricsdocs ./monitoring/metricsdocs
```

Comment:
Do we need to build and commit a separate binary which just adds documentation as a part of project. Instead of this can we just make it in a way where we execute the script and the documentation of the project gets updated. As an operator author, I would not want an additional binary which just writes/updates metrics related documentation in the project.

Reply:
changing to `go run` only